test/index.js

```diff
@@ -292,6 +292,7 @@ describe('lib', () => {
                 key: 'agencies',
                 field: 'Organization.Name',
                 type: 'facet',
+                lastUpdateTime: null,
```

Comment:
Shouldn’t this not be needed if you’re stripping nulls ?

Reply:
It does need it 🤷‍♀️

Reply:
lets dig deeper 😄

Reply:
According to our conversation, we'll make the tests lest rigid :)